### ISSUE TITLE
line 221 of inria3.py ask for more than 1 return value.

### ISSUE THREAD

**Issue by dragon9001** [lixiang-ucas/Building-A-Nets#1]
The line 221 of inria3.py states:

`G_logist, G, G_var = build_fc_densenet(z, preset_model = args.model, num_classes=num_classes, is_bottneck=True, compression_rate=0.5)`
However, This build_fc_densenet() function returns only one tensor, the code at the end of this function looks like:
`      #####################
      #      Softmax      #
      #####################
      net = slim.conv2d(stack, num_classes, [1, 1], scope='logits')
      return net`

which only returns one tensor, how shall I change the code to make it runable?

**Comment by lixiang-ucas** [comment]
code corrected.
